### ISSUE TITLE
Error building bootstrap-sass with 2.10.2

### ISSUE THREAD

**Issue by craiglyons** [brunch/sass-brunch#141]
I get the following error building `bootstrap-sass` latest (`3.3.7`) when using `sass-brunch` latest (`2.10.2`).
Downgrading to `2.10.1` fixes the problem.  I'll try to dig in when I have some more time in case this isn't actually an issue with `sass-brunch`. 

```
error: Invalid mapping: {"generated":{"line":1432,"column":28},"source":"node_modules/bootstrap-sass/assets/stylesheets/bootstrap/_type.scss","original":{"line":286,"column":-1},"name":null}
```

**Comment by paulmillr** [comment]
I released 2.10.3 which mitigates this temporarily. Will investigate more in the meantime.

**Comment by paulmillr** [comment]
@craiglyons can you help me reproduce the issue? Here's the repo with 2.10.2 and bootstrap-sass — all fine. I've tried Ubuntu and MacOS.
